pom.xml

```diff
@@ -105,6 +105,12 @@
       <artifactId>vertx-web</artifactId>
       <version>${vertx.version}</version>
     </dependency>
+    <dependency>
```

Comment:
Please change this dependency to mod-audit-client https://repository.folio.org/#browse/browse:maven-snapshots:org%2Ffolio%2Fmod-audit-client%2F0.0.5-SNAPSHOT%2F0.0.5-20200906.165513-1.